portfolio/src/main/webapp/SlideShow.js

```diff
@@ -0,0 +1,78 @@
+// Copyright 2019 Google LLC
```

Comment:
I dont think you need to include all this copyright stuff but like no harm, i guess